pkgs/development/interpreters/python/cpython/3.6/default.nix

```diff
@@ -64,6 +64,14 @@ in stdenv.mkDerivation {
     substituteInPlace configure --replace '-Wl,-stack_size,1000000' ' '
   '';
 
+  patches = optionals (!(stdenv.cc.isGNU or false)) [
```

Comment:
@FRidh Just saw your post on the 2.7 version of this issue: https://github.com/NixOS/nixpkgs/pull/19585#discussion_r83555826 what's your take on `!isGNU`?